rules/travis.js

```diff
@@ -74,15 +75,23 @@ module.exports = ctx => {
 				});
 			}
 
-			const versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+			let versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+
+			versions = versions.filter(v => IGNORED_VERSIONS.indexOf(v) === -1);
 
 			for (const version of versions) {
 				if (DEPRECATED_VERSIONS.indexOf(version) !== -1) {
 					ctx.report({
 						message: `Version \`${version}\` is deprecated.`,
 						file
 					});
-				} else if (version !== 'node' && engine && !semver.satisfies(normalize(version), engine)) {
+				}
+			}
+
+			versions = versions.filter(v => DEPRECATED_VERSIONS.indexOf(v) === -1);
```

Comment:
We would then have to filter out the deprecated versions again in the [`if (engine)` block](https://github.com/SamVerschueren/clinton/blob/v0.11.0/rules/travis.js#L94-L104) further down. [Like you said](https://github.com/SamVerschueren/clinton/pull/64#issuecomment-276000160):

> In order to ignore node decently, I'd suggest just using filter on the versions array so we don't have to take that one into account further down in the code.